modules/ROOT/pages/how-a-bdm-is-deployed.adoc

```diff
@@ -205,44 +205,3 @@ classloader (and in the process classloaders of all processes on this tenant), a
 Then the new BDM is directly available, even if in most cases, new (versions of) processes must be (re-)deployed to take
 advantage of the BDM.
 
-
```

Comment:
row 177: the limitations that we refer to have been moved to another page.
We should create a reference to this chapter in the "bdm-management-in-bonita-bpm-portal" page.

Reply:
Yes, I'll do that.

Reply:
Done